.github/workflows/ci.yml

```diff
@@ -75,5 +75,25 @@ jobs:
         file: ./coverage.xml
         yml: ./codecov.yml
 # Disabled this line because Codecov has been extra flaky lately, and having to re-run the entire
-# test suite until every coverage upload step succeeds is more of a hassle than it's worth. 
+# test suite until every coverage upload step succeeds is more of a hassle than it's worth.
 #        fail_ci_if_error: true
+  upload:
+    runs-on: ubuntu-latest
+#    if: github.event_name == 'schedule'
```

Comment:
I just thought of something else regarding this conditional.

Do we need to specify the branch name (`master`)?

Reply:
Based on the other `schedule` workflow runs, it seems to use the "default" working branch, so I think this is probably fine.